packages/actor-context-preprocess-query-source-identify/lib/ActorContextPreprocessQuerySourceIdentify.ts

```diff
@@ -54,6 +58,26 @@ export class ActorContextPreprocessQuerySourceIdentify extends ActorContextPrepr
         .map(querySource => this.expandSource(querySource)));
       const querySources: IQuerySourceWrapper[] = await Promise.all(querySourcesUnidentifiedExpanded
         .map(async querySourceUnidentified => this.identifySource(querySourceUnidentified, action.context)));
+
+      /**
+       * When identifying sources in preprocess actor, we record this as a dereference seed document event
+       */
+      const statistics: IStatisticsHolder = action.context.get(KeysInitQuery.statistics)!;
+      const statisticDereferenceLinks = <IStatisticBase<ILink>> statistics
```

Comment:
This should also work:
```suggestion
      const statisticDereferenceLinks: IStatisticBase<ILink> = statistics
```